src/Tasks/ComReference.cs

```diff
@@ -407,21 +407,31 @@ internal static string StripTypeLibNumberFromPath(string typeLibPath, FileExists
         private static string GetModuleFileName(IntPtr handle)
         {
             bool success = false;
```

Comment:
nit: The `success` local is not needed anymore.

Reply:
Done.